docs/exts/includes/dag-definition.rst

```diff
@@ -0,0 +1,32 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. dag-definition-start
+A DAG is a model that encapsulates everything needed to execute a workflow. Some DAG attributes include the following:
```

Comment:
I thought we want to change it to `dag` here as Daniel suggested.

Reply:
I don't think it makes sense to change to `dag` in the documentation when there are 100s of other instances of `DAG`. IMO, in a separate PR, we can change all references in docs to `dag`